firebase-auth.html

```diff
@@ -45,6 +45,12 @@
 
 The `user` property will automatically be populated if an active session is
 available, so handling the resolved promise of sign-in methods is optional.
+
+It's important to note that if you're using a Service Worker you should let
+urls that contain `/__auth/` go through to the network, rather than have the
```

Comment:
URLs that start with `/__/` should be the actual pattern, as the entire `__` namespace is reserved for Firebase.